pkg/pulsar/token.go

```diff
@@ -84,10 +84,18 @@ func (t *token) CreateSecretKey(signatureAlgorithm algorithm.Algorithm) ([]byte,
 func (t *token) Create(algorithm algorithm.Algorithm, signKey interface{}, subject string,
 	expireTime int64) (string, error) {
 
-	claims := &jwt.MapClaims{
-		"sub": subject,
-		"exp": jwt.NewNumericDate(time.Unix(expireTime, 0)),
+	var claims *jwt.MapClaims
+	if expireTime == 0 {
```

Comment:
Do you need to check if it is less than 0?

Reply:
Done.